lxd/device/nic_bridged.go

```diff
@@ -612,7 +619,7 @@ func (d *nicBridged) allocateFilterIPs(n *network.Network) (net.IP, net.IP, erro
 
 	// Check DHCPv6 is enabled on parent if dynamic IPv6 allocation is needed.
 	if shared.IsTrue(d.config["security.ipv6_filtering"]) && IPv6 == nil && !canIPv6Allocate {
-		return nil, nil, fmt.Errorf("Cannot use security.ipv6_filtering as DHCPv6 is disabled or no IPv6 on parent %s and no static IPv6 address set", d.config["parent"])
+		logger.Warn("IPv6 filtering enabled without a static IPv6 specified or dynamic allocation enabled, all IPv6 traffic will be blocked", log.Ctx{"project": d.inst.Project(), "instance": d.inst.Name(), "dev": d.name})
```

Comment:
I'm tempted to say this should go to Debug if not entirely removed.
Environments (thinking of Travis) that run thousands of containers with such a setup will get tired of the warning on every startup :)

Reply:
@stgraber ok will remove.